ci/azure-pipeline-security-cloudfoundrysinglesignon.yaml

```diff
@@ -1,18 +0,0 @@
-trigger:
```

Comment:
The file `ci/update-pipelines.sh` contains logic based on "CloudFoundrySingleSignon", does it need to be adapted?

Reply:
That is now dead code, but it won't break anything. I am not sure that we will need the UAA pieces when we revisit integration testing this sample, but I'm also not sure yet that we won't